src/resources/views/layout.blade.php

```diff
@@ -102,7 +102,7 @@
 
     <!-- jQuery 2.2.0 -->
     <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
-    <script>window.jQuery || document.write('<script src="{{ asset('vendor/adminlte') }}/plugins/jQuery/jQuery-2.2.0.min.js"><\/script>')</script>
+    <script>window.jQuery || document.write('<script src="{{ asset('vendor/adminlte') }}/plugins/jQuery/jQuery-2.2.3.min.js"><\/script>')</script>
```

Comment:
Ideally if we're going to do this, we'd actually look through the plugins directory for something that "looked like" jQuery although this does work, I suppose.

Reply:
or better: don't rely on AdminLTE and include it to backpack itself.

Reply:
https://thecodingmachine.github.io/discovery/

I don't really recomment it - it just seems to add even more complexity.

Reply:
I don't think there is a need for something like `discovery`. Just making sure the file is still there when updating AdminLTE or including it to backpack itself will do.